Modules/_sqlite/cache.c

```diff
@@ -263,7 +263,7 @@ static PyMethodDef cache_methods[] = {
 
 PyTypeObject pysqlite_NodeType = {
         PyVarObject_HEAD_INIT(NULL, 0)
-        MODULE_NAME "Node",                             /* tp_name */
+        MODULE_NAME ".Node",                             /* tp_name */
```

Comment:
This looks like the sperate issue.
Please open the new issue for this and revert the change :)

Thanks alot!

Reply:
Fixed in 21e678871e28d11c3a84a6380b540756f7ff7a7f